recipes/indexlr.cpp

```diff
@@ -29,6 +29,7 @@ const static size_t INITIAL_OUTPUT_STREAM_SIZE = 100;
 const static size_t QUEUE_SIZE = 64;
 const static size_t MAX_THREADS = 5;
 const static size_t DEFAULT_THREADS = MAX_THREADS;
+const static size_t MAX_QUALITY = 32;
```

Comment:
Set this as I misunderstood something about Phred qualities, and I can remove it from the code but:

I also wanted to set an upper bound for this value, as if the user sets it too high, it'll bee to restrictive and many good quality minimizers will be replaced by a fake minimizer of higher quality adjacent to them (something user may not know much about)... a good upper bound would probably be much lower than 32 tho... what do you think?

Reply:
just did:
```
"  --qual      Include minimizer Phred quality string in the output"
```

Reply:
Hmm good point about the 'fake' minimizer - and do we have a condition to avoid choosing any 'max' minimizers? 

I'm just unsure about having a hard max altogether - could it be something like if they set it above say 30 (Q30) that we print a warning message? Just thinking that it isn't out of the question that we could have long reads with that quality or higher in the future (ex. Illumina reads can have qualities of up to at least 40)

Reply:
Yeah no we already skip any minimizer that its value is `max`. By fake, I meant a kmer that should not have been picked but been picked because other kemrs adjacent to it were disqualified by their quality... and as we talked on Friday, such a  kmer would not generate a useful minimizer.

Yeah that's what I thought first, but would they really see/care the warnings? but sure I'll switch to a warning.

Reply:
Yeah just thinking if we're trying to be more general purpose, avoiding (potentially arbitrary) upper bounds is probably a good idea? Since it's a code library - so we'd leave the decision up to the user. Definitely possible that a user wouldn't read the warnings, but at least that's a compromise where we're giving the user the power/information